### ISSUE TITLE
Excel Export bug

### ISSUE THREAD

**Issue by ZiedHf** [telerik/kendo-react#58]
**Bug report**
- Exporting Excel file does not works as expected.

**Current behavior**
- Looping an array 'columns' inside the <Grid> exports an empty Excel file.

:stop_sign:  This code does not work : 
```
<ExcelExport
          data={dataSource}
          ref={exporter => {
            this._export = exporter;
          }}
        >
          <Grid data={dataSource}>
            <GridToolbar>
              <button title="Export PDF" className="k-button k-primary" onClick={this.export}>
                Export to Excel
              </button>
            </GridToolbar>
            {/* <Column field="tat" title="TATATAT" />
            <Column field="row_serial" title="row_serial" />
            <Column field="ordered" title="ordered" /> */}
            {columns &&
              columns.map((column, key) => (
                <Column
                  field={column.field}
                  title={column.title}
                  key={key}
                />
              ))}
          </Grid>
        </ExcelExport>
```
:+1:  This one works as expected : 
```
<ExcelExport
          data={dataSource}
          ref={exporter => {
            this._export = exporter;
          }}
        >
          <Grid data={dataSource}>
            <GridToolbar>
              <button title="Export PDF" className="k-button k-primary" onClick={this.export}>
                Export to Excel
              </button>
            </GridToolbar>
            <Column field="tat" title="TATATAT" />
            <Column field="row_serial" title="row_serial" />
            <Column field="ordered" title="ordered" />
          </Grid>
        </ExcelExport>
```

## Expected behavior
- Should work in both cases.

## Environment

Package versions:
- @progress/kendo-react-excel-export: 1.2.0
- @progress/kendo-react-grid: 1.2.0

**Comment by Xizario** [comment]
Until we fix this, you could use the [excel export package directly](https://www.telerik.com/kendo-react-ui/components/excelexport/), it will work for dynamically set columns.
An example: https://stackblitz.com/edit/react-9ab8lv?file=app/main.js

**Comment by digego** [comment]
any progress on fixing this?

**Comment by simonssspirit** [comment]
@digego we have researched it and started working on it.

Once it is in the develop version we will update the issue.

**Comment by nstoychev** [comment]
Fixed in version 2.8.0-dev.201903151331.
Here is an updated example: https://stackblitz.com/edit/react-7xddct?file=app/main.js

**Comment by Xizario** [comment]
Fixed in `2.8.0`
